test/GPv2Encoding.test.ts

```diff
@@ -221,11 +152,13 @@ describe("GPv2Encoding", () => {
 
       // NOTE: `v` must be either `27` or `28`, so just set it to something else
       // to generate an invalid signature.
-      const encodedTradeBytes = ethers.utils.arrayify(encoder.encodedTrades);
-      encodedTradeBytes[285] = 42;
+      const trades = encoder.trades;
+      const signature = ethers.utils.arrayify(trades[0].signature);
+      signature[64] = 42;
```

Comment:
Finally we won't have to update this number every time we change the encoding!

Reply:
:tada: